src/routes.js

```diff
@@ -3,15 +3,19 @@ import React from 'react'
 import AddVmButton from './components/VmDialog/AddVmButton'
 import PageRouter from './components/PageRouter'
 import { VmDetailToolbar, PoolDetailToolbar } from './components/Toolbar/index'
-import VmsPage from './components/Pages/VmsPage'
```

Comment:
I don't really like removing `VmsPage` component. Because all pages must be in the same place. If I want to find Detail page or Vms page I always know that their in Pages directory. Thus I think it better to return `VmsPage` component, maybe not to separate file, but at least to `Pages/index.jsx`

Reply:
Great point - I'll put that back in `Pages/index.js`